style.css

```diff
@@ -0,0 +1,117 @@
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    overflow: auto;
+}
+
+.logo {
+    height: 50%;
+    width: 10%;
+    display: block;
+    border: solid black 1px;
+}
+
+h1 {
+    margin: 0;
+}
+
+#one {
+    border-top: solid black 1px;
+    overflow: auto;
+}
+
+#two {
+    border-top: solid black 1px;
+    overflow: auto;
+}
+
+#three {
+    border-top: solid black 1px;
+}
+
+#four {
+    border-top: solid black 1px;
+}
+
+nav {
+    position: fixed;
+    top: 0;
+    background: paleturquoise;
+    height: 8%;
+    opacity: 0.5;
+}
+
+ul{
+    list-style-type: none;
+    text-align: center;
+    padding: 1% 0% 0% 0%;
+}
+li {
+    display: inline;
+    padding: 0% 5% 0% 5%;
+}
+
+.container {
+    width: 50%;
+    margin: auto;
+    overflow: auto;
+    border: solid black 1px;
+}
+
+.about-me-text {
+    text-align: left;
+    width: 500px;
+    float: left;
+    border: solid black 1px;
+    word-wrap: break-word;
+}
+
+.photo {
+    width: 200px;
+    float: right;
+    border: solid black 1px;
+}
+
+.links {
+    width: 25%;
+    height: 40%;
+    padding: 15px;
+    border: solid black 1px;
+}
+
+.width {
+    width: 100%;
+}
+
+.alignment {
+    margin-left: auto;
```

Comment:
can do margin-left and margin-right in one line eg: margin: 0 auto;

Reply:
Done

Reply:
Done